cupy/random/distributions.py

```diff
@@ -283,6 +283,32 @@ def normal(loc=0.0, scale=1.0, size=None, dtype=float):
     return x
 
 
+def pareto(a, size=None, dtype=float):
+    """Pareto II or Lomax distribution.
+
+    Returns an array of samples drawn from the pareto II distribution. Its
+    probability density function is defined as
+
+    .. math::
+        f(x) = \\alpha(1+x)^{-(\\alpha+1)},
+
+    Args:
+        a (float): Parameter of the gamma distribution :math:`\\alpha`.
+        size (int or tuple of ints): The shape of the array. If ``None``, a
+            zero-dimensional array is generated.
```

Comment:
It seems `a` can be a `cupy.ndarray` and `size=None` will generate an output with the same size as `a`.